kubric/core/base.py

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# TODO(klaus): shouldn't this logic be moved to util.py, and this file renamed to asset.py?
+
```

Comment:
I'm in favor of renaming, but I don't see any merit in moving `next_global_count` to utils, since this is the only place we use this function. We could rename it to `_next_global_count` and move it to the bottom of the file though.

Reply:
Let me check, I thought next_global_cound is used elsewhere.